src/main/java/io/quarkus/search/app/indexing/IndexingService.java

```diff
@@ -231,65 +218,23 @@ private void indexAll(FailureCollector failureCollector) {
         Log.info("Indexing...");
         try (Rollover rollover = Rollover.start(searchMapping)) {
             try (QuarkusIO quarkusIO = fetchingService.fetchQuarkusIo(failureCollector)) {
-                indexQuarkusIo(quarkusIO);
+                Log.info("Indexing quarkus.io...");
+                searchMapping.scope(Object.class).massIndexer()
+                        .batchSizeToLoadObjects(indexingConfig.batchSize())
+                        .context(QuarkusIOLoadingContext.class, QuarkusIOLoadingContext.of(quarkusIO))
+                        // this could lead to a BlockedThreadChecker complaining if called through /reindex
+                        //  [i.ve.co.im.BlockedThreadChecker] Thread Thread[vert.x-worker-thread-1,5,main] has been blocked for 61076 ms, time limit is 60000 ms:
```

Comment:
This is weird, we probably need to raise a limit somewhere.

Or make this reactive... which is technically possible, but I don't really want to see what the try-with-resources would look like in a reactive pipeline :x

Reply:
Strange that we've never seen it before 🤷🏻‍♂️🤔 

> Or make this reactive...

👀 👀 👀 👀 maybe not today 🙈 😃